test/upload-test.js

```diff
@@ -25,10 +26,16 @@ describe('http-api', function() {
   before(function(done) {
     tempDir = path.join(os.tmpdir(), 'probo-asset-receiver-' + Date.now());
     var options = {
-      databaseDataDirectory: tempDir,
-      fileDataDirectory: tempDir,
+      fileStoragePlugin: 'awsS3Storage',
```

Comment:
Tests should be using file storage. For testing the AWS plugin we would want to use [nock](https://www.npmjs.com/package/nock) or similar. Did we run the tests after changing this? If so it seems they must have failed given the current `createReadStream()` implementation in the aws storage plugin.